app/threads/client/threads.css

```diff
@@ -97,11 +116,13 @@
 		white-space: nowrap;
 		text-overflow: ellipsis;
 
+		align-items: center;
+
 		color: var(--rc-color-alert-message-primary);
 
-		align-items: center;
+		margin: 0 0.25rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "margin" to come before "color" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5416119)